go/test/endtoend/reparent/plannedreparent/reparent_test.go

```diff
@@ -379,9 +379,15 @@ func TestReparentDoesntHangIfPrimaryFails(t *testing.T) {
 
 	// Perform a planned reparent operation, the primary will fail the
 	// insert.  The replicas should then abort right away.
+	ver, err := getVTExecVersion("vttablet")
+	require.NoError(t, err)
 	out, err := utils.Prs(t, clusterInstance, tablets[1])
-	require.Error(t, err)
-	assert.Contains(t, out, "primary failed to PopulateReparentJournal")
+	if ver <= 15 {
+		require.Error(t, err)
+		assert.Contains(t, out, "primary failed to PopulateReparentJournal")
+	} else {
+		require.NoError(t, err, "PRS should not fail.")
+	}
```

Comment:
I don't understand why we have the version checking when that is generally driven by the version of vitess that the test changes are in.

Reply:
in downgrade test we use the same test but with lower version of vttablet which now no longer fail during reparent insert when the schema is changed.